Tests/RxSwiftTests/PrimitiveSequence+DebugTest.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  PrimitiveSequence+DebugTest.swift
+//  Rx
+//
+//  Created by muukii on 3/13/17.
+//  Copyright © 2017 Krunoslav Zaher. All rights reserved.
+//
+
+import XCTest
+import RxSwift
+import RxTest
+
+class PrimitiveSequenceDebugTest : RxTest {
+
+}
+
+// MARK: debug
+extension PrimitiveSequenceDebugTest {
+  func testDebug_Completed() {
```

Comment:
We would just want to test that `debug` operator will return Single type and single element.

You can look at `func testSingle_map_producesSingleElement()` for content.

The reason why we aren't implementing other tests is because we are just reusing operators from normal observable sequence.

The only new thing we need to test is that if it receives one element, it will produce one element also.

We could theoretically add full suite of unit tests for those operators also, but that would simply cause explosion of unit tests and code, and don't think we would have much benefit, and would need to maintain another huge amount of code.